makefile

```diff
@@ -0,0 +1,36 @@
+-include .env.local
```

Comment:
I've come to dislike makefiles. Generally they are little more than a list of bash aliases, and when they *do* grow beyond that, they become unreadable. IMO a `release.sh` is sufficient for most projects.